### PR TITLE
Implement NFS helper functions

### PR DESCRIPTION
**Pull request by meldafrawi**
longhorn/longhorn#1483

### CODE REVIEW

File: manager/integration/tests/backupstore.py [hunk 1]
```diff
@@ -181,12 +207,19 @@ def backupstore_write_backup_cfg_file(client, core_api, volume_name, backup_name
...
 
-# TODO: implement nfs_write_backup_cfg_file
-def nfs_write_backup_cfg_file():
-    raise NotImplementedError
+def nfs_write_backup_cfg_file(client, volume_name, backup_name, data):
+    nfs_backup_cfg_file_path = nfs_get_backup_cfg_file_path(client,
+                                                            volume_name,
+                                                            backup_name)
+    cfg_file = open(nfs_backup_cfg_file_path, 'w')
+    cfg_file.write(json.dumps(data))
```
Comment (joshimoo) [inline manager/integration/tests/backupstore.py:221]: In the minio case you use str(), I assume the data is a python object that should still be json encoded?
```
        tmp_bkp_cfg_file.write(str(backup_cfg_data))
```

Comment (meldafrawi) [reply manager/integration/tests/backupstore.py]: Fixed

**Comment by yasker** [comment]
There is a conflict.

**Comment by joshimoo** [comment]
@meldafrawi can you rebase and fix the conflict?

**Comment by meldafrawi** [comment]
Conflict fixed, Thanks @yasker @joshimoo

**Pull request merged**